app/src/lib/git/reflog.ts

```diff
@@ -63,3 +63,47 @@ export async function getRecentBranches(
 
   return [...names]
 }
+
+/**
+ * Gets the distinct list of branches that have been checked out after a specific date
+ *
+ * @param repository the repository who's reflog you want to check
+ * @param afterDate the minimum date a checkout has to occur
+ */
+export async function getCheckoutsAfterDate(
```

Comment:
I'm not sure if this is coming in a follow-up PR, but aside from the tests this isn't being called by the main app.

Is that worth doing in this PR, given it's feature-flagged?

Reply:
Yea, you're probably right. Just added it to the main prune logic.